bot/cogs/sudo.py

```diff
@@ -40,77 +45,64 @@ def get_uptime(self) -> str:
         minutes, seconds = divmod(rem, 60)
         days, hours = divmod(hours, 24)
         if days:
-            return f"{days} days, {hours} hr, {minutes} mins, and {seconds} secs"
+            str = f"{days} days, {hours} hr, {minutes} mins, and {seconds} secs"
         else:
-            return f"{hours} hr, {minutes} mins, and {seconds} secs"
+            str = f"{hours} hr, {minutes} mins, and {seconds} secs"
 
-    async def is_owner(ctx: Context) -> t.Union[bool, None]:
-        if ctx.author.id in config.devs:
-            return True
-        else:
-            embed = Embed(description="This is an owner-only command, you cannot invoke this.", color=Color.red())
-            await ctx.send(embed=embed)
+        return str
 
     @group(hidden=True)
-    @check(is_owner)
     async def sudo(self, ctx: Context) -> None:
         """Administrative information."""
         pass
```

Comment:
```suggestion
```
You don't need it (the docstring is enough)
You can also do this :
```suggestion
        await ctx.send_help("tag global")
```